psm_utils/io/idxml.py

```diff
@@ -194,7 +194,7 @@ def _parse_psm(
             score=peptide_hit.getScore(),
             precursor_mz=peptide_id.getMZ(),
             retention_time=peptide_id.getRT(),
-            # NOTE: ion mobility will be supported by OpenMS in the future
+            ion_mobility=float(peptide_id.getMetaValue("IM")) if peptide_id.getMetaValue("IM") is not None else None
```

Comment:
A comma is needed here, and the walrus might be interesting here
```suggestion
            ion_mobility=float(im) if (im := peptide_id.getMetaValue("IM")) is not None else None,
```